chain/app/app.go

```diff
@@ -147,7 +147,7 @@ func NewApp(
 	baseAppOptions ...func(*baseapp.BaseApp),
 ) *App {
 	app := &App{}
-	app.setPlugins()
+	app.setPlugins(logger)
```

Comment:
Did you consider adding a logger field to the App struct?

Reply:
yeah - it gets set in the embedded `runtime.App`. i'm opting to just pass it directly than move things around. router needs to be setup first thing